app/controllers/searches_controller.rb

```diff
@@ -159,7 +159,7 @@ def mutual_friends
   def common_friends
     @user_items = build_user_items(@searched_tw_user.common_friends(current_user.twitter_user))
     @graph = @searched_tw_user.common_friends_graph(current_user.twitter_user)
-    @tweet_text = view_context.common_friends_text(@user_items.slice(0, 3).map{|i| i[:target] }, @searched_tw_user, @user_items.size - 3)
+    @tweet_text = view_context.common_friends_text(@user_items.slice(0, 3).map { |i| i[:target] }, @searched_tw_user, @user_items.size - 3)
```

Comment:
__[rubocop]__
* Line is too long. [126/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)